### PR TITLE
ae04a9 ae681f ae5f50

### PR DESCRIPTION
**Pull request by dziban303**
## Describe your changes

Added ae04a9 and ae681f, with images, and AE5F50, without. AE681F is a P-8 tactical hex but was [most recently used by 169567](https://twitter.com/SR_Planespotter/status/1679489909516820480)—this aircraft also uses AE5F50, as it did ten hours ago. Which hex the photos should go into is sort of a dilemma, but I opted to put them into the hex it was most recently using. 

Would like @Phaeton 's take on the image question. Should I add the same images to both hexes maybe? Or neither? Or the other one? Ugh

AE04A9 is an Army Citation.

----
# Final content
* ae681f = tactical code, no pics
* ae5f50 = 169567, with pics
* ae04a9 = army c560, with pics
* ae68b3 = usaf wolfhound, with pics

----

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Comment by Phaeton** [comment]
@dziban303 I would put the image on AE5F50

**Comment by dziban303** [comment]
@Phaeton done

**Comment by Phaeton** [comment]
Question, should AE5F50 have the reg as 169567 vs tactical?

**Comment by dziban303** [comment]
@Phaeton done. also added pics for 🐺🐶 ae68b3. I'm done with this PR

File: plane-alert-db.csv [hunk 1]
```diff
@@ -13033,4 +13033,7 @@ AE60EC,160490,United States Marine Corps,Beech T-34C Turbo Mentor,T34T,Mil,Train
 AE6C01,86-0127,United States Air Force,Rockwell B-1B Lancer,B1,Mil,Cold Warrior,Strategic Bomber,Bone,USAF,https://w.wiki/3kLm
 AE6BD3,85-0064,United States Air Force,Rockwell B-1B Lancer,B1,Mil,Cold Warrior,Strategic Bomber,Bone,USAF,https://w.wiki/3kLm
 06A248,A7-MAA,Qatar Emiri Air Force,C-17A Globemaster III,C17,Mil,Tactical Airlift,Globemaster,Touch The Sky With Glory,Other Air Forces,https://w.wiki/4ntJ
-AE1BFF,04-08706,United States Army,Boeing-Vertol CH-47F Chinook,H47,Mil,Choppa,Wocka Wocka,Twin Rotor,Toy Soldiers,https://en.wikipedia.org/wiki/Boeing_CH-47_Chinook
\ No newline at end of file
+AE1BFF,04-08706,United States Army,Boeing-Vertol CH-47F Chinook,H47,Mil,Choppa,Wocka Wocka,Twin Rotor,Toy Soldiers,https://en.wikipedia.org/wiki/Boeing_CH-47_Chinook
+AE04A9,99-00100,United States Army,Cess UC-35A Citation,C560,Mil,VIP Transport,Bizjet,Must Be Nice,Toy Soldiers,https://www.army.mil
```
Comment (Phaeton) [inline plane-alert-db.csv:13037]:
```suggestion
AE04A9,99-00100,United States Army,Cessna UC-35A Citation,C560,Mil,VIP Transport,Bizjet,Must Be Nice,Toy Soldiers,https://www.army.mil

```

**Review by Phaeton — CHANGES_REQUESTED** [review]
Found a typo on AE04A9's Type

**Comment by dziban303** [comment]
Weird. Not sure how I managed to do that.

**Pull request merged**